PRASCore.jl/src/Systems/assets.jl

```diff
@@ -45,6 +45,17 @@ Base.getindex(g::G, idxs::AbstractVector{Int}) where {G <: Generators} =
     G(g.names[idxs], g.categories[idxs],
       g.capacity[idxs, :], g.λ[idxs, :], g.μ[idxs, :])
 
+function Base.show(io::IO, g::Generators)
+    println(io, "$(length(g.names)) generators with unique categories: $(join(unique(g.categories), ", "))")
+    println(io, "\nItems:")
+    println(io, "  names: $(typeof(g.names))")
```

Comment:
I'm not sure we need to print names and categories here. I'm also not sure what new information the user is learning by printing type and size of other fields. I'm not opposed but the users should know the type of fields of assets, right?

Reply:
Agreed, there isn't any new information here. If you wanted to provide more details, name/category + some summary statistics for each property might be useful, at the risk of the output being overly long